msg/protos/sds.proto

```diff
@@ -21,6 +21,16 @@ message RspGetPPList{
   Result result = 2;
 }
 
+message ReqGetPPStatus {
+  PPBaseInfo my_address = 1;
+
+}
+
+message RspGetPPStatus{
+  uint32 ActivationState = 1;
+  Result result = 2;
+}
+
```

Comment:
we can't change this protobuf definition here as it will make currently running now incompatible with each ot

add them to the end of file.

Reply:
these two messages now moved to the end of proto file